src/extractor/extractor_callbacks.cpp

```diff
@@ -175,6 +175,38 @@ void ExtractorCallbacks::ProcessWay(const osmium::Way &input_way, const Extracti
         }
     }
 
+    const auto classStringToMask = [this](const std::string &class_name) {
+        auto iter = classes_map.find(class_name);
+        if (iter == classes_map.end())
```

Comment:
We need this for indexing: The number of elements in the classes map gives us a continuous ID per class.